docs/docs/tooling/testing.md

```diff
@@ -76,4 +76,76 @@ fn test_king_arthur() {
 fn test_bridgekeeper() {
     main(32);
 }
-```
\ No newline at end of file
+```
+
+### Fuzz tests
+
+You can write fuzzing harnesses that will run on `nargo test` by using the decorator `#[test]` with a function that has arguments. For example:
+
+```rust
+#[test]
+fn test_basic(a: Field, b: Field) {
+    assert(a + b == b + a);
+}
+```
+The test above is not expected to fail. By default, the fuzzer will run for 10 seconds and use 100000 executions (whichever comes first). All available threads will be used for each fuzz test.
```

Comment:
But then this should say it's 1 second, right?

Reply:
You're right